frontend/src/ui/components/Header/Header.tsx

```diff
@@ -39,22 +67,63 @@ export const Header: React.FC = () => {
               </NavLink>
             </Route>
             <Route path={ROUTES.Console.HOME}>
-              <NavLink className="nav-link" to={ROUTES.Console.HOME}>
+              <NavLink
+                className="nav-link"
+                to={ROUTES.Console.HOME}
+                isActive={(match, location) => {
+                  // Make customize link active for any customization pages
+                  return location.pathname.startsWith(ROUTES.Console.HOME);
+                }}
+              >
                 Customize
               </NavLink>
-              <Nav.Link onClick={() => window.open(CONTACT_US_LINK, '_blank')}>
-                Support Request
-              </Nav.Link>
-              <NavLink className="nav-link" to={ROUTES.Console.NOTICE_BOARD}>
+
+              {lmsLink && (
+                <Nav.Link
+                  href={lmsLink}
+                  target="__blank"
+                  className="nav-lms-link"
+                >
+                  {deploymentPill}
+                  <span> My site (LMS) </span>
+                  <i className="fa fa-link" style={{ fontSize: '16px' }} />
```

Comment:
Why are we styling this inline?

Reply:
It was just a small change and I forgot to move it to style.css. Now it has been moved.